misc/loop/Makefile

```diff
@@ -1,44 +1,20 @@
-# The startup delay (waits until the node is "ready")
-DELAY ?= 10 # seconds
-# The temporary backup file for transactions
-BACKUP_FILE ?= $(abspath ./txs_backup.log)
-# The entire txs history across all iterations
-HISTORY_OUTPUT := $(abspath ./txs_history.log)
+all: docker.start
 
-# The gnoland binary
-gnoland_bin := go run github.com/gnolang/gno/gno.land/cmd/gnoland
-# The tx archive binary
-tx_bin := go run github.com/gnolang/tx-archive/cmd
+docker.start: # Start the portal loop
+	docker compose up -d
 
-# The relative gno.land directory
-gnoland_dir := $(abspath ../../gno.land)
+docker.stop: # Stop the portal loop
+	docker compose down
```

Comment:
this should also stop the container running `gnoland`, no?

Reply:
it does